.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    target-branch: "v1.x"
```

Comment:
Does this mean that we're not targeting master as well?

Reply:
This means, the dependabot will only create PRs to `v1.x` branch.

The dependabot.yml that is going to master will create PRs to `master` branch